### PR TITLE
Ursula Domain/Config Parameter updates

### PR DESCRIPTION
**Pull request by derekpierre**
Updates based on https://github.com/nucypher/nucypher/pull/3271.

### CODE REVIEW

**Comment by derekpierre** [comment]
Tested via updates to `tapir` nodes.

**Comment by derekpierre** [comment]
Tested updated lynx nodes

**Review by KPrasch — APPROVED** [review]

File: nucypher_ops/playbooks/recover_ursula_ops_data.yml
Comment (manumonti) [inline nucypher_ops/playbooks/recover_ursula_ops_data.yml:71]: what is the purpose of line #78?

```
           _domain:                {{ursula_config.domain}}\n
```

I don't fully understand what this `ursula_config.domain` stands for, but it seems like it hasn't any relation with TACo domains (tapir, lynx...), so it can be confussing.

Also, it seems like this is not very used?

Maybe we could remove that line to avoid confussion or use it as the actual TACo domain.

WDYT?

**Review by manumonti — APPROVED** [review]
LGTM! ✌️

I left a little comment, but it is an unrelated one.

So we can merge this PR and address it, if applicable, later on.

File: nucypher_ops/playbooks/recover_ursula_ops_data.yml
Comment (derekpierre) [reply nucypher_ops/playbooks/recover_ursula_ops_data.yml:71]: @manumonti apologies, I didn't see this comment before merging.

> I don't fully understand what this ursula_config.domain stands for, but it seems like it hasn't any relation with TACo domains (tapir, lynx...), so it can be confusing.

It's exactly that i.e. the TACo domain. It's the `domain` value stored in the `ursula.json` file on the remote machine. It's value can be `tapir`, `lynx`, `mainnet`.

This recovery playbook basically goes to the remote machine and gathers all of the information about the Ursula to regenerate the `nucypher-ops` configuration files. It's the remedy from when I did `rm -rf` on the config folder. Typically nucypher-ops has config files that it uses to connect and configure remote nodes. This is the opposite - there is no config file, but we connect to the machine to get the information to regenerate the configuration file.

Comment (manumonti) [reply nucypher_ops/playbooks/recover_ursula_ops_data.yml:71]: got it! Thanks

**Pull request merged**
